### DISCUSSION TITLE
What's XXX in the wiki comparison table?

### DISCUSSION THREAD

**Discussion by berezovskyi** [oracle/opengrok#4279] [Q&A]
https://github.com/oracle/opengrok/wiki/Comparison-with-Similar-Tools has a number of fields marked XXX, what does that mean?

**Comment by vladak** [comment]
These are to-be-done placeholders, someone put various tools to the table without doing the work of identifying the features. Will remove these.

**Reply by vladak** [reply]
Speaking of the table, it sort of alludes of competition with other tools, however at this stage of the software life cycle, OpenGrok is not trying to compete with any tool. That said, the table might be good for quick overview of what features are available. It is missing some modern tools (e.g. Sourcegraph) and some properties (e.g. horizontal scaling, Docker image, etc.).
